src/components/Global/InvitesGraph/index.tsx

```
@@ -1136,11 +1156,76 @@ export default function InvitesGraph(props: InvitesGraphProps) {
+                        ctx.globalAlpha = alpha
+                        ctx.font = 'bold 5px Inter, system-ui, -apple-system, sans-serif'
+                        ctx.fillStyle = '#fbbf24' // Gold color
+                        ctx.textAlign = 'center'
+                        ctx.textBaseline = 'middle'
+                        ctx.fillText('+1 point', arrival.x, arrival.y + yOffset)
+                        ctx.restore()
+                    }
+                })
+
+                // Clean up expired arrivals
+                toDelete.forEach((linkId) => arrivals.delete(linkId))
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)